src/data/polynomial/degree/definitions.lean

```diff
@@ -640,6 +648,14 @@ lemma monic.ne_zero_of_ne (h : (0:R) ≠ 1) {p : R[X]} (hp : p.monic) :
   p ≠ 0 :=
 by { nontriviality R, exact hp.ne_zero }
 
+lemma monic_of_nat_degree_le_of_coeff_eq_one (pn : p.nat_degree ≤ n) (p1 : p.coeff n = 1) :
+  monic p :=
```

Comment:
What I meant was also having
```lean
lemma monic_of_degree_le_of_coeff_eq_one (pn : p.degree ≤ n) (p1 : p.coeff n = 1) :
  monic p :=
```

Reply:
Ah, ok!  I added this lemma as well now!